Source/WebKit/WebProcess/Plugins/PDF/PDFPluginChoiceAnnotation.mm

```diff
@@ -52,22 +55,20 @@
     PDFPluginAnnotation::updateGeometry();
 
     RefPtr styledElement = downcast<StyledElement>(element());
-    styledElement->setInlineStyleProperty(CSSPropertyFontSize, choiceAnnotation().font.pointSize * plugin()->contentScaleFactor(), CSSUnitType::CSS_PX);
+    styledElement->setInlineStyleProperty(CSSPropertyFontSize, annotation().font.pointSize * plugin()->contentScaleFactor(), CSSUnitType::CSS_PX);
 }
 
 void PDFPluginChoiceAnnotation::commit()
 {
-    choiceAnnotation().stringValue = downcast<HTMLSelectElement>(element())->value();
+    annotation().widgetStringValue = downcast<HTMLSelectElement>(element())->value();
 
     PDFPluginAnnotation::commit();
 }
 
 Ref<Element> PDFPluginChoiceAnnotation::createAnnotationElement()
 {
     Ref document = parent()->document();
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    PDFAnnotationChoiceWidget *choiceAnnotation = this->choiceAnnotation();
-ALLOW_DEPRECATED_DECLARATIONS_END
+    PDFAnnotation *choiceAnnotation = annotation();
```

Comment:
`RetainPtr`

Reply:
Fixed in [116c479](https://github.com/WebKit/WebKit/commit/116c47933b9948f1c2e49242b9a652d27f3dee49)